### PR TITLE
Tasks for adding and deleting modules

### PR DESCRIPTION
**Pull request by tahliar**
Rendered PDFs:

[task-add-modules-extensions-after-installation_color_en.pdf](https://github.com/SUSE/doc-modular/files/7018999/task-add-modules-extensions-after-installation_color_en.pdf)

[task-delete-modules-extensions_color_en.pdf](https://github.com/SUSE/doc-modular/files/7019002/task-delete-modules-extensions_color_en.pdf)

### CODE REVIEW

**Comment by tahliar** [comment]
`-list-extensions` is what's in the Deployment Guide (and it works), but `--list-extensions` is what's in the SUSEConnect help output, and is a more standard convention, so I've updated it.

File: xml/task-add-modules-extensions-after-installation.xml [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    <para>&slehpc;&nbsp;15&nbsp;SP3, 15&nbsp;SP2, 15&nbsp;SP1, 15&nbsp;GA</para>
+   </listitem>
+   <listitem>
+    <para>&sled;&nbsp;15&nbsp;SP3, 15&nbsp;SP2, 15&nbsp;SP1, 15&nbsp;GA, 12&nbsp;SP5, 12&nbsp;SP4, 12&nbsp;SP3</para>
+   </listitem>
+   <listitem>
+    <para>&slert;&nbsp;15&nbsp;SP3, 15&nbsp;SP2, 15&nbsp;SP1, 15&nbsp;GA, 12&nbsp;SP5, 12&nbsp;SP4, 12&nbsp;SP3</para>
+   </listitem>
+  </itemizedlist>
+ </section>
```
Comment (lproven) [inline xml/task-add-modules-extensions-after-installation.xml:68]: Gosh that's a big list!

At first I was going to suggest making it hierarchical but on a 2nd reading, how about listing versions and products separately?

This applies to versions 12 SP3 through to 15 SP3 of:
[prod list]

Or even 

This applies to: [product list]
And these versions [version list]

Something like that. It's horribly repetitive to my eyes.

I welcome others' opinions on this.

File: xml/task-add-modules-extensions-after-installation.xml [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+ <section xml:id="introduction-add-modules-extensions-after-installation">
+  <title>Introduction</title>
+   <para>
+    Even after a system is installed and registered, you can still add modules
+    and extensions to your system, either with &yast; or from the command line.
+    All extensions require additional registration codes which are liable for cost.
```
Comment (lproven) [inline xml/task-add-modules-extensions-after-installation.xml:75]: "Liable _for_ cost" is clumsy.

"Which may incur additional licence costs"?

File: xml/task-add-modules-extensions-after-installation.xml [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+       <imagedata fileref="yast2_extension_list_sles.png"
+                  width="100%" format="png"/>
+       </imageobject>
+     </mediaobject>
+    </informalfigure>
+   </step>
+   <step>
+    <para>
+     Select all modules and extensions you want to install. Note that all
+     extensions require additional registration codes, which are liable for cost.
```
Comment (lproven) [inline xml/task-add-modules-extensions-after-installation.xml:133]: See above comment.

File: xml/task-add-modules-extensions-after-installation.xml [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+   </step>
+   <step>
+    <para>
+     Click <guimenu>Next</guimenu>.
+    </para>
+   </step>
+   <step>
+    <para>
+     If you are prompted to agree to a license, review the terms of the license,
+     then activate <guimenu>I Agree to the License Terms</guimenu> and click
```
Comment (lproven) [inline xml/task-add-modules-extensions-after-installation.xml:153]: "Activate" seems clumsy. "Click"? "Select"? "Pick" or "Choose"?

**Review by lproven — CHANGES_REQUESTED** [review]

File: xml/task-add-modules-extensions-after-installation.xml (hunk 4 above)
Comment (tahliar) [reply xml/task-add-modules-extensions-after-installation.xml:153]: Normally I would say "Select", but our Style Guide says to use "Activate" for check boxes.

File: xml/task-add-modules-extensions-after-installation.xml (hunk 1 above)
Comment (tahliar) [reply xml/task-add-modules-extensions-after-installation.xml:68]: This is a (temporary) part of the templates, so any changes would need to be applied to the templates as well. 

I think it's in list form for metadata reasons, but @taroth21 will know more.

**Pull request merged**